package.json

```diff
@@ -26,6 +27,7 @@
         "@xstate/test": "^0.5.1",
         "cypress": "^10.3.0",
         "dialog-polyfill": "^0.5.6",
+        "directory-tree": "^3.3.1",
```

Comment:
same with dependency - we can have it in gh-actions env, meaning we can fetch in in the worklfow and don't need it as UI/Vite dependency - pls remove this line